packages/muelu/src/Utils/MueLu_Monitor.cpp

```diff
@@ -4,6 +4,7 @@ int MueLu::FactoryMonitor::timerIdentifier_=0;
 namespace MueLu {
   PrintMonitor::PrintMonitor(const BaseClass& object, const std::string& msg, MsgType msgLevel) : object_(object) {
     tabbed = false;
+    Kokkos::Profiling::pushRegion(msg);
```

Comment:
Ah, I wasn't seeing any MueLu tracing regions in my analysis, and couldn't find a flag to enable it, so I just dropped this into the Factory/SubMonitor via the PrintMonitor. I can remove this line since it feels a bit out of place. Is there a flag/env-var that enables kokkos tracing in MueLu?

Reply:
You could try `Teuchos_KOKKOS_PROFILING=ON`.

Reply:
@seanofthemillers I've been able to get traces before.  Do have the Kokkos profiling tools enabled?

Reply:
I have removed this line. I have the `Teuchos_KOKKOS_PROFILING` on and tested with the roctx tracer from the kokkos-tools repo, but was not able to get MueLu factories to show up in the trace (I had lots of Belos and Tpetra regions though). I'm probably missing something that enables kokkos profiling in Teuchos's timers - I'll look at that the next chance I get.

Reply:
Ah, figured it out. I needed to set the verbosity flag to "timings" in the preconditioner file in order to enable the Teuchos stacked timer in the TimeMonitor, which then triggers the kokkos profiling regions.